<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEntryWidgetAdvanced</class>
 <widget class="QWidget" name="EditEntryWidgetAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>366</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="attributesBox">
     <property name="title">
      <string>Additional attributes</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="AttributesListView" name="attributesView"/>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="attributesEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="addAttributeButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editAttributeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeAttributeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="attachmentsBox">
     <property name="title">
      <string>Attachments</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListView" name="attachmentsView">
        <property name="flow">
         <enum>QListView::LeftToRight</enum>
        </property>
        <property name="isWrapping" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="addAttachmentButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeAttachmentButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openAttachmentButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveAttachmentButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AttributesListView</class>
   <extends>QListView</extends>
   <header>gui/entry/EditEntryWidget_p.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>attributesView</tabstop>
  <tabstop>attributesEdit</tabstop>
  <tabstop>addAttributeButton</tabstop>
  <tabstop>editAttributeButton</tabstop>
  <tabstop>removeAttributeButton</tabstop>
  <tabstop>attachmentsView</tabstop>
  <tabstop>addAttachmentButton</tabstop>
  <tabstop>removeAttachmentButton</tabstop>
  <tabstop>saveAttachmentButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
